src/modules/dynamicDisplay.js

```diff
@@ -0,0 +1,172 @@
+import updateLikes from './updateLikes.js';
+import addComment from './addComment.js';
+import showCounter from './showCounter.js';
+import getComments from './getComments.js';
+
+export default class dynamicDisplay {
+generateCard = (data, cardId, cardUniqueId, showImage, showName) => {
+  const container = document.getElementById('container');
+
+  const id = cardId + 1;
+  const show = document.createElement('div');
+  show.classList.add('show');
+  const poster = document.createElement('img');
+  poster.classList.add('poster');
+  poster.src = showImage;
+  const top = document.createElement('div');
+  top.classList.add('top');
+  const title = document.createElement('h3');
+  title.classList.add('title');
+  title.innerText = showName;
+  const likes = document.createElement('div');
+  likes.classList.add('likes');
+  const span = document.createElement('span');
+  span.classList.add('like');
+  span.classList.add('material-icons-outlined');
+  span.innerText = 'favorite';
+  span.setAttribute('id', id);
+
+  const counter = document.createElement('p');
+  counter.classList.add('counter');
+
+  const options = document.createElement('div');
+  options.classList.add('options');
+  const commentBtn = document.createElement('button');
+  commentBtn.setAttribute('id', id);
+  commentBtn.innerText = 'Comment';
+  const reservationBtn = document.createElement('button');
+  reservationBtn.setAttribute('id', id);
+  reservationBtn.innerText = 'Reservation';
+
+  show.appendChild(poster);
+  show.appendChild(top);
+
+  top.appendChild(title);
+  top.appendChild(likes);
+
+  likes.appendChild(span);
+  likes.appendChild(counter);
+
+  show.appendChild(options);
+  options.appendChild(commentBtn);
+  options.appendChild(reservationBtn);
```

Comment:
- [ ] Since the reservation was not implemented, I think it would be a good idea to remove the `Reservation button`